front/app/containers/Admin/pages-menu/containers/HeroBanner/messages.ts

```diff
@@ -109,19 +109,19 @@ export default defineMessages({
     defaultMessage: 'Customise the hero banner image and text.',
   },
   heroBannerSaveButton: {
-    id: 'app.containers.Admin.pagesAndMenu.heroBannerSaveButton',
+    id: 'app.containers.Admin.pages-menu.heroBannerSaveButton',
     defaultMessage: 'Save hero banner',
   },
   heroBannerButtonSuccess: {
-    id: 'app.containers.Admin.pagesAndMenu.heroBannerButtonSuccess',
+    id: 'app.containers.Admin.pages-menu.heroBannerButtonSuccess',
```

Comment:
I don't think it really matters, but I wonder if we should change all instances of `pages-menu` to `PagesAndMenu` or `pagesAndMenu` 

In the translation file we have `app.containers.Admin.Moderation` and `"app.containers.Admin.Users.GroupsHeader` and in the app directory we generally have things like `CookiePolicy` and `EmailSettingsPage`


It's different because under the Admin module all of the subsections are lowercase and one word:

![Screen Shot 2022-07-27 at 13 58 50](https://user-images.githubusercontent.com/3614128/181241433-1fc5527d-13b2-42e0-89c8-b1380e06daf0.png)

In the end I don't think it matters really. @jinjagit thoughts on this?

Reply:
Yes, agree. Sounds like `PagesAndMenu` would be more conventional.

Reply:
I think in that case it would be `pagesAndMenu` in the folder directory and `PagesAndMenu` in the translation file. So I think just a find/replace in `en.json` and then changing the folder name in VSCode (which should offer to fix all imports automatically). 
 
The routes can stay under `pages-menu`